### DISCUSSION TITLE
Is there a character limit in inky, or is this a bug?

### DISCUSSION THREAD

**Discussion by wmbenedetto** [inkle/ink#726] [Writing in ink]
I'm just getting started trying out ink using inky, and I'm running into an issue right from the start. I don't see anything in the docs about this, but it appears there is a character limit beyond which text no longer shows up?

As you can see here, the text and options are displayed in the preview window as expected. 
<img width="1434" alt="Screen Shot 2021-12-16 at 11 16 46 PM" src="https://user-images.githubusercontent.com/250323/146505061-841fc759-d7ff-41aa-a8cc-aaa5fca99ed9.png">

But if my opening paragraph is long, it no longer shows up in the preview. 
<img width="1432" alt="Screen Shot 2021-12-16 at 11 17 07 PM" src="https://user-images.githubusercontent.com/250323/146505064-0b83970e-6003-4ef5-8537-062b4bf0ac74.png">

By trial and error, I figured out that it shows up to 243 characters. Beyond that, the text disappears.
<img width="1434" alt="Screen Shot 2021-12-16 at 11 17 24 PM" src="https://user-images.githubusercontent.com/250323/146505066-07567f84-4ab4-4e64-986d-14f209adcc4f.png">

The issue only seems to apply to the first paragraph. Subsequent paragraphs can be long with no problems.
<img width="1432" alt="Screen Shot 2021-12-16 at 11 58 40 PM" src="https://user-images.githubusercontent.com/250323/146510045-1be1efed-f61f-4fbd-b4d7-0b38520139d1.png">

Selecting Option 1 with this text works and displays option_1 as expected ...
![image](https://user-images.githubusercontent.com/250323/146510583-5bcb193c-746d-480c-9734-0a94d4257369.png)

But if the first paragraph of option_1 is one word, it drops the entire second paragraph ...
![image](https://user-images.githubusercontent.com/250323/146510794-a3c9c880-96b4-4d7d-adf9-775e133d0ff1.png)

Am I doing something wrong, or is this just a bug in inky?

**Comment by GreenCloversGames** [comment]
Couldn't replicate this, however I am on a Windows computer. Looks like it is a bug of some sort, I would put this on the Issues page of Inky : [https://github.com/inkle/inky/issues](https://github.com/inkle/inky/issues)
